thirdPartySrc/analysisServer/org/dartlang/analysis/server/protocol/FlutterOutline.java

```diff
@@ -0,0 +1,322 @@
+/*
+ * Copyright (c) 2015, the Dart project authors.
```

Comment:
Instead of adding another top-level directory, could thirdPartySrc/analysisServer be moved into third_party? It also needs to have an Intellij "source" directory defined (see third_party/intellij-plugins-dart/src) in the module definition. That will correct the build failure.

The header comment looks like it should be updated -- not sure on the rules, but we're not doing Eclipse now. That applies to all files in thirdPartySrc.